tools/heir-opt.cpp

```diff
@@ -461,6 +499,7 @@ void mlirToBgvPipelineBuilder(OpPassManager &pm,
   // Secretize inputs
   pm.addPass(createSecretize(SecretizeOptions{options.entryFunction}));
   pm.addPass(createWrapGeneric());
+  pm.addPass(createConvertIfToSelect());
```

Comment:
if the data oblivious transforms are being added to the bgv pipeline, maybe all the passes should be run via a call to `convertToDataObliviousPipelineBuilder(pm);`?